pghoard/webserver.py

```diff
@@ -313,6 +316,25 @@ def _prefetch(self, site, filetype, names):
                 with suppress(Exception):
                     os.unlink(tmp_target_path)
 
+    def get_status(self, site):
+        try:
+            state_file_path = self.server.config["json_state_file_path"]
+            if site is None:
+                if os.path.exists(state_file_path):
+                    with open(state_file_path, 'r') as fp:
+                        state_json_data = fp.read()
+                    raise HttpResponse(state_json_data, status=200)
+                else:
+                    raise HttpResponse(status=404)  # Not Found
+            else:
+                # TODO: Handle site specific status
+                # I suggest sending 406 if the site doesn't exist.
+                # Right now we will return 400 because this isn't implmented yet
+                raise HttpResponse(status=400)  # Bad Request
+        except (IOError, OSError, TypeError):
```

Comment:
Please remove this exception handler - the request will be handled in the ` _response_handler` context manager which will turn unhandled exceptions into HttpResponses (status 503).  Once that's fixed I'll be happy to merge this, sorry about the slow responses to your pull request.